src/plugins/QuestScripts/QuestHooks.cpp

```diff
@@ -178,10 +178,10 @@ void DeeprunRatRoundup(Player* pPlayer, Object* pObject)
 	(TO_CREATURE(pObject))->SendChatMessage(CHAT_MSG_MONSTER_SAY, LANG_UNIVERSAL, "Into the box me pretties! Thas it. One by one ye go.");
 }
 
-void MaybellComplete(Player* pPlayer, Object* pObject)
+void TheEscape(Player* pPlayer, Object* pObject)
 {
-	(TO_CREATURE(pObject))->SendChatMessage(CHAT_MSG_MONSTER_YELL, LANG_UNIVERSAL, "Here goes nothing...");
-	(TO_CREATURE(pObject))->Emote(EMOTE_ONESHOT_CRY);
+	(TO_CREATURE(pObject))->SendChatMessage(CHAT_MSG_MONSTER_SAY, LANG_UNIVERSAL, "Here goes nothing...");
+	(TO_CREATURE(pObject))->SetOrientation(5.39307);
```

Comment:
This triggers a warning for me:

> ..\..\..\src\src\plugins\QuestScripts\QuestHooks.cpp(184): warning C4305: 'argument' : truncation from 'double' to 'float'

Please fix it.
( You just have to add an f after the number, like 5.39307f )